go.mod

```diff
@@ -0,0 +1,81 @@
+module github.com/ethereum/go-ethereum
+
+go 1.12
+
+require (
+	bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898
+	github.com/Azure/azure-pipeline-go v0.0.0-20180607212504-7571e8eb0876 // indirect
+	github.com/Azure/azure-storage-blob-go v0.0.0-20180712005634-eaae161d9d5e
+	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6 // indirect
+	github.com/allegro/bigcache v0.0.0-20190218064605-e24eb225f156
+	github.com/apilayer/freegeoip v0.0.0-20180702111401-3f942d1392f6
+	github.com/aristanetworks/goarista v0.0.0-20170210015632-ea17b1a17847
+	github.com/btcsuite/btcd v0.0.0-20171128150713-2e60448ffcc6
+	github.com/cespare/cp v0.1.0
+	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/davecgh/go-spew v1.1.1
+	github.com/deckarep/golang-set v0.0.0-20180603214616-504e848d77ea
+	github.com/docker/docker v0.0.0-20180625184442-8e610b2b55bf
+	github.com/edsrzf/mmap-go v0.0.0-20160512033002-935e0e8a636c
+	github.com/elastic/gosigar v0.0.0-20180330100440-37f05ff46ffa
+	github.com/fatih/color v1.6.0
+	github.com/fjl/memsize v0.0.0-20180418122429-ca190fb6ffbc
+	github.com/gballet/go-libpcsclite v0.0.0-20190403181518-312b5175032f
```

Comment:
This should be upgraded to the latest version, which is `2772fd86a8ff4306d2749f610a386bfee9e0d727`